src/DatePicker/DatePicker.test.js

```diff
@@ -298,33 +292,11 @@ describe('<DatePicker />', () => {
     });
 
     describe('onBlur callback', () => {
-        test('should call onBlur after clicking outside calendar overlay', () => {
-            const blur = jest.fn();
-            const element = mount(<DatePicker onBlur={blur} />);
-
-            element.find('button.fd-button--transparent.sap-icon--calendar').simulate('click');
-
-            element.find('table.fd-calendar__table tbody.fd-calendar__group tr.fd-calendar__row td.fd-calendar__item:not(.fd-calendar__item--other-month)')
-                .at(0)
-                .simulate('click');
-
-
-            //trigger onBlur by clicking outside
-            simluateOutsideMouseClick();
-
-            expect(blur).toHaveBeenCalledTimes(1);
-
-            expect(blur).toHaveBeenCalledWith(expect.objectContaining({ date: expect.any(moment) }));
-        });
```

Comment:
Don't think this test/story is necessary, and will cause validation issues if onBlur runs validation prematurely because of an outside click.